src/Tests/JsonRegression.fs

```diff
@@ -201,4 +201,66 @@ let tests =
             }
             compareResourcesToJson [ vwan ] "virtual-wan.json"
         }
+        
+        test "LoadBalancer" {
+            let myVnet =
+                vnet {
+                    name "my-vnet"
+                    add_address_spaces [ "10.0.1.0/24" ]
+                    add_subnets [
+                        subnet {
+                            name "my-services"
+                            prefix "10.0.1.0/24"
+                            add_delegations [
+                                SubnetDelegationService.ContainerGroups
+                            ]
+                        }
+                    ]
+                }
+            let lb =
+                loadBalancer {
+                    name "lb"
+                    sku Farmer.LoadBalancer.Sku.Standard
+                    add_frontends [
+                        frontend {
+                            name "lb-frontend"
+                            public_ip "lb-pip"
+                        }
+                    ]
+                    add_backend_pools [
+                        backendAddressPool {
+                            name "lb-backend"
+                            link_to_vnet "my-vnet"
+                            load_balancer "lb"
```

Comment:
We shouldn't (ideally) need to specific the name of the load balancer - it's contained by the parent anyway.